packages/utils/src/miscellany.ts

```diff
@@ -40,11 +40,11 @@ export function mangleScopedPackage(packageName: string): string {
 }
 
 export function removeVersionFromPackageName(packageName: string | undefined): string | undefined {
-  return packageName?.replace(/\/v(\d){1,}$/i, "");
+  return packageName?.replace(/\/v\d+(\/\*)?$/, "$1");
```

Comment:
Remove the case insensitive flag cause version directories must start with a lowercase `v`.